cmd/ethrex_l2/src/commands/utils.rs

```diff
@@ -0,0 +1,28 @@
+use clap::Subcommand;
+use keccak_hash::H256;
+use secp256k1::SecretKey;
+
+#[derive(Subcommand)]
+pub(crate) enum Command {
+    #[clap(about = "Convert private key to address.")]
+    PrivateKeyToAddress {
+        #[arg(long = "pk", help = "Private key in hex format.", required = true)]
```

Comment:
I find positional arguments easier and more natural to use in commands like this one. What do you think?
```suggestion
        #[arg(help = "Private key in hex format.", required = true)]
```